storybookjs/storybook
# issue #4174 : support the tagConstructor option

Issue:

## What I did
added the support for configuring a specific tagConstuctor in a story
See #4174

## How to test

Is this testable with Jest or Chromatic screenshots? yes
Does this need a new example in the kitchen sink apps? yes
Does this need an update to the documentation? no

If your answer is yes to any of these, please make sure to include it in your PR.
